src/Microsoft.DotNet.TestFramework/project.json

```diff
@@ -14,7 +14,7 @@
     "NETStandard.Library": "1.5.0-rc2-23922"
   },
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.5": {
       "imports": "dnxcore50"
```

Comment:
It's broken to be importing "dnxcore50" from a "netstandard1.5" library.  That problem needs to be fixed across the repo but not in this change.